server-setup/setting-reset-bindings.md

```diff
@@ -113,6 +113,33 @@ Make sure OVR Advanced Settings is closed before following these steps or you wi
 <iframe width="100%" height="auto" src="https://www.youtube.com/embed/KuCjmHBpH7E" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
 
+### OVR Toolkit
+{:.no_toc}
+
+If you do not wish to bind keyboard shortcuts directly to your controller buttons, OVR Toolkit's "wristwatch" has Macro functionality that can be used to trigger Resets and Quick Resets.
+
+Demo GIF:
+
+![img](..\assets\img\ovrtDemo.gif)
+
+1. Open the OVR Toolkit settings. It may be in the [system tray.](..\assets\img\ovrtTray.webp)
+2. Click on "Settings" on the top, then "Setup Wrist Macros" on the right.
+3. Click on "Macro Icons" at the bottom, this will open a folder that you can place PNGs for custom macro icons. Download and copy these two icon files for the resets into there. [Reset Icon.](..\assets\img\resetBold.png) [Quick Reset Icon.](..\assets\img\quickresetBold.png)
+4. Close and re-open OVR Toolkit, otherwise the icons won't show up yet.
+5. Click "Add Macro", and click "Add key press" until you have enough for each key of the binding. (Default needs 4).
+6. For each key in sequence, click "Rebind" and then push one of the binding's buttons once. Having the letter as the bottom key is important, as that is the order the keys are pressed. Entering `Y+CTRL+ALT+SHIFT` would not work, but `CTRL+ALT+SHIFT+Y` would.
+7. Click the blank icon square on the left side of the entry, and choose the corresponding icon. (To differentiate, Reset is the basic reset symbol. Quick Reset has the flame in the middle.)
+8. Repeat steps 5-7 for the other binding.
+9. Click "Save Changes", and then check the Macros tab on your OVR Toolkit wristwatch for your new bindings! (You may need to restart OVR Toolkit!)
+
+Example GIF of adding the Quick Reset binding:
+
+![img](..\assets\img\ovrtMacro.gif)
+
+#### Warning
+
+Currently the SlimeVR server resets the positions the instant the binding is pressed, which is not ideal if you are looking down at your extended arms. However, this can be overcome with a simple AutoHotKey script until a configuration option is added to add a countdown to those bindings. [AutoHotKey Script Here](https://gist.github.com/nullstalgia/18595feddac83ad8d6d8e0e8117e64d1)
```

Comment:
It would be cool if you could add the Gist's code here directly in a code block

Reply:
Agree with Anna

Reply:
Yep, I can absolutely do that! I just was not sure what would look best. If you guys don't edit it first, I'll get to it soon

Reply:
Done! I'll let one of you guys resolve this.